src/Chainweb/Miner/RestAPI.hs

```diff
@@ -55,6 +55,10 @@ type MiningApi_ =
     :<|> "mining" :> "updates"
                   :> ReqBody '[OctetStream] ChainBytes
                   :> Raw
+    :<|> "mining" :> "stream"
+                  :> Capture "chain" ChainId
+                  :> Capture "miner" MinerId
+                  :> Raw
```

Comment:
Implicit `servant` behaviour: When no `ReqBody` is required here, `Raw` defaults to `GET`. If there is instead a `ReqBody`, it becomes `POST`.